### ISSUE TITLE
在restaurant, laptop数据集报错 could not broadcast input array from shape (25) into shape (300)

### ISSUE THREAD

**Issue by CHENWEI19980314** [songyouwei/ABSA-PyTorch#19]

**Comment by songyouwei** [comment]
请贴上详细的错误栈信息

**Comment by songyouwei** [comment]
> 问一下最后测试准确率test-acc是怎么算的，为什么还有算一个最大的max-acc,f1有什么用呢

测试准确率的计算请参考 https://github.com/songyouwei/ABSA-PyTorch/blob/e4f069fcb5f28b7f045034e631aaeccf7ad425c9/train.py#L103

max_test_acc 是用来记录并保存最好的模型结果的，用于inference

**Comment by CHENWEI19980314** [comment]
在论文中出现的准确率在您的代码里是哪一个呢？

**Comment by CHENWEI19980314** [comment]
twitter数据集好像有误，我看很多论文上面都没有用这个数据集，为什么呢

**Comment by songyouwei** [comment]
acc=accuracy, f1=f1 score
有些论文用到了twitter数据集

**Comment by CHENWEI19980314** [comment]
您是怎么控制迭代次数的？20epoch是人为设定的吗？repeat是交叉训练吗？

**Comment by songyouwei** [comment]
20epoch是个经验值，这个你也可以自己根据数据集的训练情况调整。repeat是重置所有参数后重新训练，以寻求可能的更优的参数。

**Comment by CHENWEI19980314** [comment]
损失函数在这个有什么用呢？代码中保存在state_dict文件夹下最高的就是该模型最后的准确率吗？

**Comment by songyouwei** [comment]
损失函数的作用。。这个你应该去看书 :joy:
state_dict里acc最高的通常就是学到的最好的模型

**Comment by CHENWEI19980314** [comment]
感谢大佬指点

**Comment by CHENWEI19980314** [comment]
问一个问题，如果我要在word embedding后面加上position embedding的信息，应该怎么加呢？

**Comment by songyouwei** [comment]
> 问一个问题，如果我要在word embedding后面加上position embedding的信息，应该怎么加呢？

参考 https://github.com/songyouwei/ABSA-PyTorch/blob/840962d1ed47a08398cdeda1bfbdf208db7c5995/models/ram.py#L12
